components/DetailList.tsx

```diff
@@ -0,0 +1,110 @@
+import { useState, useEffect } from 'react';
+import styled from 'styled-components';
+
+import { getEvent } from '../api/sports';
+import { EventType } from '../model/Event';
+import { Error } from './Error';
+
+const DetailBox = styled.div`
+  display: flex;
+  min-width: 50%;
+  min-height: 10%;
+  background-color: white;
+  overflow: hidden;
+`;
+
+const Header = styled.header`
+  font-size: 1.5rem;
+  font-weight: 700;
+`;
+
+const DetailContent = styled.div`
+  padding: 0rem;
+`;
+
+const List = styled.div`
+  display: flex;
+  flex-direction: column;
+  min-width: 20%;
+  background-color: ${({ theme }) => theme.color.cream};
+  background-repeat: no-repeat;
+  padding: 3rem;
+`;
+
+const ListHeaders = styled.header`
```

Comment:
Go and find semantic web guidelines, somewhere in WCAG f.e. When the header should and shouldn't be used, we have too many of those. And as you absolutely correctly adjust the web to be accessible, let's not make it worse.